src/main/java/org/lareferencia/core/harvester/workers/HarvestingWorker.java

```diff
@@ -152,7 +159,8 @@ public void run() {
 
 
 		// if fetch identify is true, will try to fetch the identify information
-		if ( runningContext.getNetwork().getBooleanPropertyValue("HARVEST_IDENTIFY_PARAMETERS") ) {
+		//if ( runningContext.getNetwork().getBooleanPropertyValue("HARVEST_IDENTIFY_PARAMETERS") ) {
```

Comment:
Will you left this here? Or it's just for testing?

Reply:
Now it does not use the property anymore, I've just commented the line for the moment, We should remove it soon.
Now it uses this.fetchIdentifyParameters, which is a worker property instead.
Please let me know if I am missing something here.